pkg/alerts/location_failure_conditions.go

```diff
@@ -0,0 +1,96 @@
+package alerts
+
+import (
+	"fmt"
+)
+
+// ListLocationFailureConditions returns alert conditions for a specified policy.
```

Comment:
For some of these resources we have been providing a `Get` (single) command as well that is just based on the `List` command.  Not a blocker but we might consider doing that here as well.

Reply:
For this particular API I didn't see a get method, based on the API Explorer.  I can check with the source team, but I'd left it out since the API didn't look to support it.

Reply:
I've confirmed with the team that there is not a Get api for this, but there will be in the upcoming graphql api.